lib/login.dart

```diff
@@ -27,10 +28,31 @@ class _Login extends State<Login> {
   String _server = "";
   String _username = "";
   String _password = "";
+  SharedPreferences? _prefs;
+  bool isRemeberMe = false;
+
+ 
+  GlobalKey<ScaffoldState> scaffoldKey = GlobalKey<ScaffoldState>();
 
   @override
   Widget build(BuildContext context) {
     return Scaffold(
+      key: scaffoldKey,
+      floatingActionButton: FloatingActionButton(
+        child: Icon(Icons.menu),
+        onPressed: () {
+          scaffoldKey.currentState?.openEndDrawer();
+        },
+        backgroundColor: Colors.transparent,
+      ),
+      floatingActionButtonLocation: FloatingActionButtonLocation.endTop,
+      endDrawer: Drawer(backgroundColor: Color.fromARGB(100, 0, 0, 0),
+        child: ListView(children: [
+          DrawerHeader(
+              child: Image.network(
+                  "https://avatars.githubusercontent.com/u/108163041?s=96&v=4"))
```

Comment:
I would download the image, and then use local .PNG file, instead of fetching it everytime the app launches.